src/services/chat/prepopulated.js

```diff
@@ -36,6 +36,29 @@ module.exports = async (id, users) => {
          * boleh tampil kecuali untuk user usup
          */
 
+        await chat.updatePartial({
+          set: {
+            better_channel_member: [
+              {
+                user: {
+                  ...ownUser
+                },
+                user_id: ownUser?.id,
+                is_anonymous: false,
+                updated_at: new Date().toISOString()
+              },
+              {
+                user: {
+                  ...user
+                },
+                user_id: ownUser?.id,
+                is_anonymous: false,
+                updated_at: new Date().toISOString()
+              }
+            ]
+          }
+        });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a logical error in the code. The `user_id` for both the `ownUser` and `user` objects is set to `ownUser?.id`. This could lead to incorrect data being stored if the `user` object's id is different from `ownUser?.id`. 

```diff
-                user_id: ownUser?.id,
+                user_id: user?.id,
```

Also, it would be better to check if `ownUser` and `user` are not null before spreading their properties. If either of these is null, it will throw an error.

```diff
-                  user: {
-                    ...ownUser
-                  },
+                  user: ownUser ? { ...ownUser } : {},
```

```diff
-                  user: {
-                    ...user
-                  },
+                  user: user ? { ...user } : {},
```

Lastly, consider using a transaction when updating the database and sending a message. This ensures that if one operation fails, the other won't execute, maintaining data consistency.


<!-- This is an auto-generated comment by OSS CodeRabbit -->